crates/task-impls/src/consensus.rs

```diff
@@ -365,6 +365,16 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                 }
             }
 
+            // Only vote if you has seen the VID share for this view
+            if let Some(_vid_share) = self.vid_shares.get(&proposal.view_number) {
+            } else {
+                error!(
```

Comment:
Can we reduce this to a `debug` level message?

Reply:
Changed to `info` level.